### ISSUE TITLE
Remove tags.yml files if we are sure we no longer need them

### ISSUE THREAD

**Issue by SPPearce** [nf-core/modules#6001]

**Comment by maxulysse** [comment]
yeah, not needed at all, and also the tags in the main.nf tests

**Comment by famosab** [comment]
I think by now even the module template does not include the `tags.yml` file anymore. So we can remove them. This could either be done in bulk (look at @GallVp) or just while reviewing the whole nf-test migration?

**Comment by SPPearce** [comment]
Yeah, if we aren't using them we should just remove all the tags.yml files in bulk (ignoring the CI)

**Comment by SPPearce** [comment]
This has been done.
